src/tlog-test-json-esc.c

```diff
@@ -50,9 +51,10 @@ test(const char *name, size_t exp_res_len,
     res_res_len = tlog_json_esc_buf(res_out_buf, out_len, in_ptr, in_len);
 
 #define FAIL(_fmt, _args...) \
-    do {                                                    \
-        fprintf(stderr, "%s: " _fmt "\n", name, ##_args);   \
-        passed = false;                                     \
+    do {                                                            \
+        fprintf(stderr, "FAIL %s:%d: %s " _fmt "\n", file, line,  \
```

Comment:
Please align the backslash at the end of this line horizontally with all the others in the macro.